crates/bws/src/cli.rs

```diff
@@ -11,7 +11,7 @@ pub(crate) const PROFILE_KEY_VAR_NAME: &str = "BWS_PROFILE";
 pub(crate) const SERVER_URL_KEY_VAR_NAME: &str = "BWS_SERVER_URL";
 
 pub(crate) const DEFAULT_CONFIG_FILENAME: &str = "config";
-pub(crate) const DEFAULT_CONFIG_DIRECTORY: &str = ".bws";
+pub(crate) const DEFAULT_CONFIG_DIRECTORY: &str = ".config/bws";
```

Comment:
Does this path separator work on windows ?

How about backwards compatibility ? Someone who would upgrade to this bws version, would need to start the config and access tokens setup from scratch ?

Reply:
> Does this path separator work on windows ?

Yes, I have verified the path separator does work on Windows.

> How about backwards compatibility ? Someone who would upgrade to this bws version, would need to start the config and access tokens setup from scratch ?

Great thought. As discussed we will pass on this for now.